packages/inventory-compliance/src/Utilities/FilterConfigBuilder.js

```diff
@@ -125,7 +148,11 @@ class FilterConfigBuilder {
     )
 
     getItemByLabelOrValue = (query, category) => {
-        const items = this.getCategoryForLabel(category).items;
+        const categoryConfig = this.getCategoryForLabel(category);
+        const items = categoryConfig.type !== conditionalFilterType.group ?
+            categoryConfig.items : categoryConfig.items.flatMap((item) => (
+                item.items.map((subItem) => ({ ...subItem, osMajorVersion: item.value }))
```

Comment:
Why does this say `osMajorVersion`?

Reply:
Oh, i'll change that to `parentValue`. I was switching compliance and FEC too much. :D